src/main/java/org/ow2/proactive/catalog/service/WorkflowXmlManipulator.java

```diff
@@ -78,7 +76,7 @@
             xformer.transform(new DOMSource(doc), new StreamResult(answer));
             return answer.toByteArray();
         } catch (Exception e) {
-            throw new UnprocessableEntityException(e);
+            throw new UnprocessableEntityException("It was not possible to parse object: " + e);
```

Comment:
It would be better for future to create a specific exception for parsing error instead of duplicating UnprocessableEntityException message. (duplication with KeyValueLabelMetadataHelper.java)

Reply:
I did the change